.github/pull_request_template.md

```diff
@@ -1,17 +1,28 @@
 Description
 -----------
-Describe here the proposed change. Thank you for contributing to mitiq! 🙂
 
-
-If you have not finished all tasks in the checklist, you can also open a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/) to let the others know this on-going work and keep this checklist in the PR description.
```

Comment:
Can we add the draft PR note to the tips at the end? I know that its not a well known feature but can be helpful for someone to get help/feedback before full review.

Reply:
That's a good point @crazy4pi314

Reply:
It's just under the checklist now:

![image](https://user-images.githubusercontent.com/32416820/111813852-20bfe280-88b0-11eb-97e9-2a5376f80dc1.png)

If we want to move it to tips that's fine with me.